src/styles/blocks/minute.scss

```diff
@@ -0,0 +1,11 @@
+.minute {
+  position: absolute;
+  bottom: 50%;
+  z-index: 1;
+  width: 10px;
+  height: 100px;
+  background-color: blue;
+  border-radius: 5px;
+  animation: rotate 3600s infinite;
+  transform-origin: bottom center;
```

Comment:
move minute and second classes to one file and use BEM 
also separate common functionality and rules into mixin and include it into those classes, leaving only unique rules in each class directly
```
.watch {
  &__minute {
  }
 
  &__second {
   }
}
```